dojo/tools/twistlock/parser.py

```diff
@@ -355,11 +371,20 @@ def get_item(vulnerability, test, baseImage):
         mitigated=None,
         severity_justification=f"{vector} (CVSS v3 base score: {cvss})\n\n{riskFactors}",
         impact=severity,
-        vuln_id_from_tool= vulnerability['id'],
-        publish_date=dateutil.parser.parse(vulnerability.get('publishedDate')) if vulnerability.get('publishedDate', None) else None,
-
+        vuln_id_from_tool=vulnerability["id"],
+        publish_date=(
+            dateutil.parser.parse(vulnerability.get("publishedDate"))
```

Comment:
f publishedDate is None, an exception is raised for dateutil, is correct ?

Reply:
I valid if publishedDate is None, otherwise this field allows null